pages/wishlist/wishlist.css

```diff
@@ -0,0 +1,34 @@
+h1 {
+  text-align: center;
+  margin-top: 3rem;
+}
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
+  gap: 2rem;
+  margin-top: 4rem;
+  width: 100%;
+}
+
+.grid-container .cart {
+  width: 26rem;
+  margin: auto;
+  transition: all 0.3s ease;
+}
+
+.grid-container .cart .img-container .product-action-icon li {
```

Comment:
avoid over nesting classes as it will be difficult to debug later

Reply:
I have used this way because of specificity concerns. Though curious to know how it will create problems later in debugging?

Reply:
if you want to overwrite some CSS properties, then the overwritten properties need to have same specificity as this nested one

Reply:
Noted